examples/complex_ekyc/specification.yml

```diff
@@ -5,7 +5,8 @@ user_claims:
       !sd time: "2012-04-23T18:25Z"
       !sd verification_process: f24c6f-6d3f-4ec5-973e-b0d8506f3bc7
       evidence:
-        - !sd type: document
+        - !sd
+          !sd type: document
```

Comment:
what does this double `!sd` mean?

Reply:
This is a selectively disclosable array element, which is an object containind selectively disclosable keys.